### PR TITLE
Fix Travis config and apache request auth header

### PR DESCRIPTION
**Pull request by reb3r**
**Travis config**
To get travis builds working following changes are mainly taken from core library:

- add phpunit in composer.json
- add php 7.2 to .travis.yml and remove hhvm and 5.3
- using vendor phpunit in .travis.yml
- changed superclasses in tests according to phpunit version
- removed a count() call in response to get php 7.2 working

Maybe the last point could be solved better but the solution is sufficent for given tests.

**Apache request auth header fix**

Also added commit to fix apache request auth header create from request to createFromRequest

follows up 82aec77

cleans up
TrogloGeek/oauth2-server-httpfoundation-bridge@d10366e
and follows suggestions in #20 

Basically this PR is the same as #30 but with more recent master branch as base rather than develop

### CODE REVIEW

**Comment by bshaffer** [comment]
We've moved to github actions, so this is obsolete. Thanks for your contribution!
